examples/HabitatBuilderWithS3RDS.md

```diff
@@ -0,0 +1,62 @@
+# HowTo - Setup On-Premise Depot to Leverage RDS and S3.
+
+These instructions will allow a highly-available on-premise depot installation leveraging cloud services (in this case, RDS and S3).
+## Before You Start
+
+### Assumptions
+
+* New Instance:
+  * 16 Cores
+  * 32 GB RAM
+  * /hab partitioned to a 20 GB volume
+  * /tmp partitioned to a 50 GB volume
+  * Communication opened on 443, 80, Postgres RDS port
+  * Able to get root access
+
+### Tested Versions
+
+* Chef Habitat On-Premise Depot
+
+## Setup On-Premise Depot to Leverage RDS and S3
+
+1. Download the Zip archive of the on-prem-builder repo:<br />
+`curl -LO https://github.com/habitat-sh/on-prem-builder/archive/master.zip`
+2. Download the Chef Habitat cli tool:<br />
+`curl -Lo hab.tar.gz https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-%24latest-x86_64-linux.tar.gz`
+3. From the zip archive, install the hab binary somewhere in $PATH and ensure it has execute permissions:<br />
+`sudo chmod 755 /usr/bin/hab`<br />
+`sudo hab # read the license and accept if in agreement, as the root user`<br />
```

Comment:
Consider changing this to `hab license accept` to aide scriptability

Reply:
Fixed. Good call.